sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -76,6 +76,11 @@ private[execution] sealed trait HashedRelation extends KnownSizeEstimation {
    */
   def keys(): Iterator[InternalRow]
 
+  /**
+   * Returns an iterator for values of InternalRow type.
+   */
+  def values(): Iterator[InternalRow]
```

Comment:
nit: Down below this is referred to as a build row. Should values be renamed to buildRows ? or just rows ?

Reply:
@agrawaldevesh - this is to return all `values` of a key-value pair hashed relation.